app/app/src/main/java/com/globant/calculator/android/mvp/view/CalculatorView.java

```diff
@@ -12,22 +16,49 @@
     TextView calculationLabel;
     @BindView(R.id.input_label)
     TextView inputLabel;
+    @BindView(R.id.dot_button)
+    Button dotButton;
+    @BindView(R.id.add_button)
+    Button addButton;
+    @BindView(R.id.subtraction_button)
+    Button subtractionButton;
+    @BindView(R.id.divide_button)
+    Button divideButton;
+    @BindView(R.id.multiply_button)
+    Button multiplyButton;
+    @BindView(R.id.equals_button)
+    Button equalsButton;
 
     public CalculatorView(Activity activity) {
         super(activity);
         ButterKnife.bind(this, activity);
     }
 
-    public void showResult(Integer result) {
-        calculationLabel.setText(result.toString());
+    public void showResult(String result) {
+        calculationLabel.setText(result);
     }
 
-   public void showNumberPressed(String number){
-       inputLabel.setText(number);
-   }
+    public void showNumberPressed(String number) {
+        inputLabel.setText(number);
+    }
 
-   public void showOperationPressed(String symbol){
-       inputLabel.setText(symbol);
-   }
+    public void showOperationPressed(String symbol) {
+        inputLabel.setText(symbol);
+    }
 
+    public void showError() {
+        Toast.makeText(getContext(), "Operación Invalida", Toast.LENGTH_LONG).show();
```

Comment:
no hardcode pls.. use a string resource for this message

Reply:
done, thanks.